src/components/scores/ScoreWidget.vue

```diff
@@ -0,0 +1,21 @@
+<script lang="ts" setup>
+import { ref } from "vue";
+import FairScore from "@/components/scores/FairScore.vue";
+import CareScore from "@/components/scores/CareScore.vue";
+
+const props = defineProps<{
+    score: {[key: string]: number};
+    name: string;
+}>();
+
+const collapse = ref(false); // collapse to a badge
```

Comment:
What's this state for?

Reply:
That ref was for collapsing the scores into a smaller badge on mobile, which will be implemented in a future PR. I've removed unused code in the latest commit.